.github/workflows/build-notebooks-TEMPLATE.yaml

```diff
@@ -244,6 +248,9 @@ jobs:
 
       - name: "push|schedule|workflow_dispatch: make ${{ inputs.target }}"
         run: |
+          # print running stats on disk occupancy
+          (while true; do df -h | grep "${HOME}/.local/share/containers"; sleep 30; done) &
```

Comment:
This is intentionally to be kept here or was used just for the debugging purposes?

Reply:
I want to leave it, it can be useful in the future, and it's not causing any problems with anything